src/components/base-control.js

```diff
@@ -109,14 +109,23 @@ export default class BaseControl extends PureComponent {
     }
   }
 
-  _onClick = (evt) => {
-    if (this.props.captureClick) {
-      evt.stopPropagation();
+  _onPointerUp = (evt) => {
+    if (this.props.captureDoubleClick) {
+      const {eventManager} = this._context;
+      const doubletap = eventManager.manager.get('doubletap');
```

Comment:
Why not go all-in with the destructuring here since we're already doing it above? Otherwise, should we check for null when accessing `doubletap.options.eanble`?

```suggestion
      const {options: {enable}} = eventManager.manager.get('doubletap');
```